### PR TITLE
setting up reinvent models package

### PR DESCRIPTION
**Pull request by Ashish13898**
@drugilsberg I have added a check for cuda now, please review it and let me know if any changes are required, thank you :)

### CODE REVIEW

File: reinvent_models/lib_invent/models/dataset.py [hunk 1]
```diff
@@ -74,5 +83,7 @@ def pad_batch(encoded_seqs):
     :param encoded_seqs: A list of encoded sequences.
     :return: A tensor with the sequences correctly padded.
     """
-    seq_lengths = torch.tensor([len(seq) for seq in encoded_seqs], dtype=torch.int64)  # pylint: disable=not-callable
+    seq_lengths = torch.tensor(
+        [len(seq) for seq in encoded_seqs], dtype=torch.int64
+    )  # pylint: disable=not-callable
     return (tnnur.pad_sequence(encoded_seqs, batch_first=True).cuda(), seq_lengths)
```
Comment (drugilsberg) [inline reinvent_models/lib_invent/models/dataset.py]: aren't we missing the check here?

**Review by drugilsberg — COMMENTED** [review]
I added some comments on where I think there is a missing cuda check

File: reinvent_models/lib_invent/models/dataset.py (hunk 1 above)
Comment (Ashish13898) [reply reinvent_models/lib_invent/models/dataset.py]: I think the file was not saved that's why it was not pushed. I just pushed another change, it should be in those changes now.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (drugilsberg) [inline .vscode/settings.json]: this file should not be added to git, please remove it and add `.vscode` to the .gitignore

**Review by drugilsberg — COMMENTED** [review]
Just a minor comment then we can merge.

**Pull request merged**